draft-ietf-quic-qpack.md

```diff
@@ -644,23 +647,23 @@ In this example, Base = n - 2
 
 ### Prefixed Integers
 
-The prefixed integer from Section 5.1 of [RFC7541] is used heavily throughout
-this document.  The format from [RFC7541] is used unmodified.  Note, however,
+The prefixed integer from {{Section 5.1 of RFC7541}} is used heavily throughout
+this document.  The format from {{RFC7541}} is used unmodified.  Note, however,
 that QPACK uses some prefix sizes not actually used in HPACK.
 
 QPACK implementations MUST be able to decode integers up to and including 62
 bits long.
 
 ### String Literals
 
-The string literal defined by Section 5.2 of [RFC7541] is also used throughout.
-This string format includes optional Huffman encoding.
+The string literal defined by {{Section 5.2 of RFC7541}} is also used
+throughout. This string format includes optional Huffman encoding.
 
 HPACK defines string literals to begin on a byte boundary.  They begin with a
 single bit flag, denoted as 'H' in this document (indicating whether the string
 is Huffman-coded), followed by the Length encoded as a 7-bit prefix integer, and
 finally Length bytes of data. When Huffman encoding is enabled, the Huffman
-table from Appendix B of [RFC7541] is used without modification and Length
+table from {{Section B of RFC7541}} is used without modification and Length
```

Comment:
Shouldn't this be: {{Appendix B of RFC7541}}

Reply:
No; kramdown-rfc2629 will do the right thing.